pkg/test/e2eutil/wait_util.go

```diff
@@ -67,3 +71,29 @@ func waitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace,
 	t.Logf("Deployment available (%d/%d)\n", replicas, replicas)
 	return nil
 }
+
+func waitForPod(t *testing.T, runtimeClient test.FrameworkClient, namespace, name string, retryInterval, timeout time.Duration) error {
```

Comment:
This is not an exposed function (lower case name) and you are not using it anywhere in this package so it won't be able to be used outside. If we want to add this to be used externally we should rename this to `func WaitForPod(....)`.

Reply:
@LiliC This is embarrassing. I tested it in another file and didn't make the change when I moved it. Please considered the updated changes.